template/tabs/tab.html

```diff
@@ -1,3 +1,3 @@
 <li ng-class="{active: active, disabled: disabled}">
-  <a href ng-click="select()" tab-heading-transclude>{{heading}}</a>
+  <a href ng-click="select($event)" tab-heading-transclude>{{heading}}</a>
```

Comment:
This `select` is only used internally, and the event is passed to `beforeSelect` expression which is documented.

Do you mean document the `scope.select` function in [src/tabs/tabs.js](https://github.com/larvanitis/bootstrap/blob/prevent-tab-switch/src/tabs/tabs.js#L229) using JSDoc?